packages/tools/merge-tree-client-replay/package.json

```diff
@@ -65,5 +65,14 @@
     "rimraf": "^2.6.2",
     "typescript": "~4.5.5",
     "typescript-formatter": "7.1.0"
+  },
+  "fluidBuild": {
+    "buildDependencies": {
+      "merge": {
```

Comment:
Why is this layer here? Are you envisioning something in the future?

Reply:
Yeah.   I want to convert some of the implicit per tool rules to use this style internally and explicit can merge or override in the future.